pom.xml

```diff
@@ -59,7 +59,7 @@
             <dependency>
                 <groupId>uk.ac.ebi.eva</groupId>
                 <artifactId>variation-commons-mongodb</artifactId>
-                <version>0.5-SNAPSHOT</version>
+                <version>0.6-SNAPSHOT</version>
```

Comment:
Use the same version number in `variation-commons-core` and `variation-commons-jpa` to avoid any potentially weird behaviour due to inconsistencies between versions.